site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -69,7 +69,7 @@
         "id": "hRTa3Ee15WsJ"
```

Comment:
<blockquote>"Learn more about <a href="https://www.tensorflow.org/tutorials/images/data_augmentation" target="_blank">data augmentation</a>"</blockquote><p> </p><p>Sounds like a command.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/2081/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Rephrased ("For more information, check out the...")</p>